docs/reference/upload-rights.rst

```diff
@@ -0,0 +1,120 @@
+Upload Rights
```

Comment:
```suggestion
Upload rights
```

Same comment for all headings in the doc; treat it as sentence case.
Thanks.

Reply:
Also for the sake of SEO i would suggest making the heading more explicit.

E.g. if you are trying to search for this topic would you just search "upload rights" (probably too vague) or "Kernel package upload rights"?

Our current Wiki content doesn't do great with search results.

Reply:
Hi @gagath , it may seem like a redundant step but that is why the `toctree` directive does allow you to specify a different title than the full title on the actual page.

```
.. toctree::

   Upload rights <upload-rights>
```

Yes the full title is "Upload Rights - Ubuntu kernel documentation" but it could be made clearer with a more specific title.

I'll leave this up to you to decide.

But please do update the headings to be sentence case as per the style guide.

Thanks.

Reply:
"kernel upload rights" is a lot more specific.
Thanks @gagath !